am-lib/src/test/java/uk/gov/hmcts/reform/amlib/RolePermissionTest.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.amlib;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.RESOURCE_NAME;
+import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.RESOURCE_TYPE;
+import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.ROLE_NAME;
+import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.SERVICE_NAME;
+
+public class RolePermissionTest {
+
+    private final AccessManagementService ams = new AccessManagementService("", "", "");
```

Comment:
Please rename to be just `service`. A `ams` is sort of cryptic.

Reply:
👍